pagination.go

```diff
@@ -8,16 +8,22 @@ import (
 	"github.com/google/go-querystring/query"
 )
 
+const (
+	PaginationOffset = "offset"
+	PaginationCursor = "cursor"
+)
+
 type PageCounter interface {
 	NumberOfPages() int64
 	CurrentPage() int64
 }
 
 type Paged struct {
-	TotalCount int64 `json:"-"`
-	PageCount  int64 `json:"-"`
-	Limit      int64 `json:"-"`
-	Page       int64 `json:"-"`
+	TotalCount int64  `json:"-"`
+	PageCount  int64  `json:"-"`
+	Limit      int64  `json:"-"`
```

Comment:
🟢 So we are keeping page and limit for now + adding the cursor?

Reply:
Both modes are applicable and depends on the caller.